docs/DEPLOYMENT_SETUP.md

```diff
@@ -12,39 +12,49 @@ $ npm start
 
 ## Deploying to AWS Elastic Beanstalk
 
-As a prerequisite for EB deployment, make sure you have already created your AWS environment. Some of the services used are listed below:
+Make sure you have created an AWS cloud environment as a prerequisite. Some of the services used are listed below:
 
-- Elastic Beanstalk / EC2 for hosting and deployment
-- VPC with peering to MongoDB Atlas
-- NAT Gateway (for static IP whitelisting with SingPass)
-- S3 for image and logo hosting
-- Elastic Container Registry for built Docker images
-- SES for sending emails
-- EFS for mounting files e.g. SingPass/MyInfo private keys into the `/certs` directory
-- Secrets Manager
+Infrastructure
 
-### Dockerrun.aws.json
+- AWS Elastic Beanstalk / EC2 for hosting and deployment
+- AWS Elastic File System for mounting files (i.e. SingPass/MyInfo private keys into the `/certs` directory)
+- AWS S3 for image and logo hosting, attachments for Storage Mode forms
```

Comment:
AWS Batch?

Reply:
we should specify that it is optional

Reply:
I was thinking we can leave that to the `formsg-batch-jobs` repo. Just reminded me that we should open source that...